test/test_status.c

```diff
@@ -0,0 +1,70 @@
+#include "status.h"
+
+#include <stdbool.h>
+#include <stdlib.h>
+
+#include "misc.h"
+#include "unity.h"
+
+void setup_test(void) {}
```

Comment:
Done. It is in the guide I use clang-format and it isn't supported so I have to go manually edit this everytime with sed or a different text editor sigh....